routes.yml

```diff
@@ -71,5 +71,5 @@ admin_logs:
 mockups:
     path: /mockups/{mockup}
     defaults:
-      _controller: "\\Rox\Mockups\MockupsController::showMockup"
+      _controller: "\\Rox\\Mockups\\MockupsController::showMockup"
```

Comment:
All slashes in this file were escaped except for these, which was throwing an error